src/views/Point/js/point.js

```diff
@@ -36,6 +39,17 @@ export default {
     ]),
     activeTabType () {
       return this.activeTab === 0 ? 'quiz' : 'assignment'
+    },
+    pointListEmpty () {
+      return !(this.pointList && this.pointList.length)
+    },
+    emptyStateSrc() {
+      switch (this.activeTabType) {
+        case 'quiz':
+          return 'quiz'
+        case 'assignment':
+          return 'assignment'
```

Comment:
hmm, is this duplicate with `activeTabType`? because the switch statement just return the same thing